core/controllers/base.py

```diff
@@ -425,6 +461,7 @@ def _render_exception(self, error_code, values):
                 self.render_template('pages/error/error.html')
 
     def handle_exception(self, exception, unused_debug_mode):
```

Comment:
I don't know, sorry; you'll need to investigate. Look at the callers, and also look at the superclass (webapp2?) documentation.